src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,9 +36,11 @@ public static RobotContainer getInstance() {
     }
 
     private void configureDefaultCommands() {
+        arm.setDefaultCommand(new XboxControl(arm, xboxController));
     }
 
     private void configureButtonBindings() {
+        a.onTrue(new SetShoulderAngle(arm, 30));
```

Comment:
This won't work because you have configured a default command for the arm already. What you can do is create a single default command that calls certain operations using ifs. See the `HolonomicDrive` command in the swerve-drive branch.

Reply:
Why this won't work? Unless it changed this year, when you press a button it overrides the default command, and after finishing it comes back to the default command